modules/packages.xqm

```diff
@@ -282,10 +283,12 @@ declare function packages:get-package-meta($app as xs:string, $name as xs:string
 (: should be private but there seems to be a bug :)
 declare function packages:public-repo-contents($installed as element(repo-app)*) {
     try {
-        let $url := $config:DEFAULT-REPO || "/public/apps.xml?version=" || packages:get-version() ||
+        for $pkgs in $packages:repos
```

Comment:
I think we should rename this to `$pkg` as this will be an item() from the tuple stream.

Reply:
How about `$pkg-list` I really would like to avoid using identical variable names for different things in the library